### PR TITLE
Add method to UIView to fill superview with insets

### PR DESCRIPTION
**Pull request by elland**
✨

### CODE REVIEW

**Comment by 3lvis** [comment]
O
M
G

**Comment by elland** [comment]
💃

File: Sources/UIView+Sweetness.swift [hunk 1]
```diff
@@ -20,6 +20,15 @@ public extension UIView {
         self.init()
         self.translatesAutoresizingMaskIntoConstraints = !autoLayout
     }
+
+    public func fillSuperview(with insets: UIEdgeInsets) {
```
Comment (marijnschilling) [inline Sources/UIView+Sweetness.swift:24]: Could we make the insets optional? So you could easily just set the exact same bounds? Or maybe create another method for that.

Comment (3lvis) [reply Sources/UIView+Sweetness.swift:24]: default value! 

`public func fillSuperview(with insets: UIEdgeInsets = UIEdgeInsetsZero) {`

Comment (elland) [reply Sources/UIView+Sweetness.swift:24]: Yes please!

Comment (marijnschilling) [reply Sources/UIView+Sweetness.swift:24]: even better :)

**Pull request merged**